Formula/kotlin.rb

```diff
@@ -12,6 +12,8 @@ class Kotlin < Formula
 
   bottle :unneeded
 
+  depends_on "openjdk" => :test
```

Comment:
Seeing how it's Java based I'd say this is also at runtime.

Reply:
yeah, that is what I thought (there gonna be some default java out for the runtime, but I am surprised to see the error)